build.sh

```diff
@@ -74,7 +74,7 @@ function main() {
     fi
     if [[ ! -z $( cat $update ) ]] ; then
       echo "Uploading new environment variables."
-      aws --profile herokles ssm put-parameter --type String --name /${PROJECT}/${ENV} --overwrite --value "$( jq -c . $json )"
+      aws --profile herokles ssm put-parameter --type String --name /${PROJECT}/${ENV} --overwrite --value "$( jq -s . $json )"
```

Comment:
Not sure why `-s` here. Do you expect multiple JSON objects as source? Slurping creates an array, is that not a problem later on when parsing during deploy?

Reply:
Shit, sorry, fixed, it was supposed to be `-S` to sort the keys.